docs/DeveloperGuide.md

```diff
@@ -334,42 +335,45 @@ The following activity diagram summarizes what happens when a user interacts wit
  | `***` | Frequent User | Use my up and down arrow keys to go back and forth between commands | I don't have to retype the same commands | 
  | `***` | New user | Receive a list of commands that the address book uses | I can familiarise myself with the commands and shortcuts | 
- | `***` | Frequent User | I can undo my actions | I can correct my mistakes | 
- | `***` | Expert User | Quickly update the status of multiple elderly contacts | I can efficiently manage my tasks | 
+ | `***` | Frequent User | I can undo my actions  | I can correct my mistakes             | 
+ | `***` | Expert User | Quickly update the status of multiple elderly contacts | I can efficiently manage my tasks     | 
  | `***` | New user | Try out the app with sample data | I can familiarise myself with it without worrying about the data I am playing around with | 
- | `***` | Onboarded user | Purge all sample data | The app is ready to be used | 
- | `***` | Frequent User | I can view a history of calls made to an elderly contact | I can track past interactions | 
+ | `***` | Onboarded user | Purge all sample data  | The app is ready to be used           | 
+ | `***` | Frequent User | I can view a history of calls made to an elderly contact | I can track past interactions         | 
  | `***` | Frequent User | Change the frequency of calls needed to be made for each elderly | I can track when I need to make calls | 
- | `***` | Expert User | Archive elderly contacts who are temporarily not part of the program | Keep my contact list organised | 
+ | `***` | Expert User | Archive elderly contacts who are temporarily not part of the program | Keep my contact list organised        | 
  | `***` | Frequent User | Export/Import my data across computers | If I would like to work using separate computers, e.g. a laptop and desktop, I can keep them updated | 
  | `***` | Expert User | Set multiple contacts as emergency contacts | I can call their next of kin in the event the elderly do not pick up or needs help | 
  | `***` | Frequent User | Export a specific elderly's details and history (All information) | So I can share it with another employee | 
  | `*` | Expert user | Be notified by the app when too many elderly calls are scheduled on one day | I will not overwork myself or spend too little time calling each elderly | 
  | `*` | Expert User | I can see a dashboard of my weekly or monthly call stats | I can track my productivity and ensure that I hit my KPI | 
  | `*` | Expert User | Set up automated messages for elderly contacts that are not reachable | I can have an alternative communication method | 
- | `*` | Expert User | Use AI to calculate the priority list for elderly based on their information | The correct eldelry are being prioritised  | 
+ | `*` | Expert User | Use AI to calculate the priority list for elderly based on their information | The correct eldelry are being prioritised | 
```

Comment:
looks like there is a typo on elderly here

Reply:
Thanks nice catch